.github/workflows/deploy.yml

```
@@ -0,0 +1,105 @@
+
+      
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: v1.10.3  
+      
+      - name: Change directory to Terraform directory
+        run: |
+         cd stack
+         ls -alh 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)